netplan/cli/utils.py

```diff
@@ -174,14 +174,42 @@ def get_interface_macaddress(interface):
     return link.get('addr', '')
 
 
+def get_interface_pciid(interface):  # pragma: nocover (covered in autopkgtest)
```

Comment:
question: Can we apply some test-mocking to get this net method covered by unit tests?